docs/azmcp-commands.md

```diff
@@ -942,7 +935,7 @@ azmcp sql elastic-pool list --subscription <subscription> \
                             --server <server-name>
 ```
 
-### Azure SQL Server Operations
+### Server
```

Comment:
What does Server mean here ?
seems like a typo

Reply:
SQL Server Operations reads better

Reply:
Ah thanks, it should be `### Azure SQL`. Will update in a new commit.